src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger.java

```diff
@@ -81,6 +83,7 @@ public StashBuildTrigger(
       String projectPath,
       String cron,
       String stashHost,
+      JSONObject proxySettings,
```

Comment:
Is the JSONObject type required here?
It is basically a Map<String,Object>
I would prefer to use proper, strongly typed DTOs here